neuralmonkey/nn/projection.py

```diff
@@ -19,9 +19,12 @@ def linear(inputs, size, scope="LinearProjection"):
         A tensor of shape batch x size
     """
     with tf.variable_scope(scope):
-        return tf.nn.seq2seq.linear(inputs, size, True)
+        # TODO this will be soon deprecated, should be rewrited with
+        # tf.contrib.layers.fully_connected and a support for list of tensors
```

Comment:
jestli `tf.contrib.layers.fully_connected` neumí list of tensors, tak si tu projekci uděláme tady sami. Podle mě neni důvod to přebírat z contrib, protože to je natolikk jednoduchá operace, že se vyplatí mít to tady, aby se každej mohl podívat na ten kód.. Už jsi viděl `tf.contrib.layers.fully_connected`?